esmvaltool/interface_scripts/interface.ncl

```diff
@@ -0,0 +1,339 @@
+; This is a temporary file updated by Python on the fly
```

Comment:
This comment does not apply anymore, right?

Reply:
Definitely not!

Reply:
I've removed the outdated comment. Maybe you can write a better summary of the functionality provided by the file?

Reply:
It's a big topic, so I made an [issue](https://github.com/ESMValGroup/ESMValTool/issues/182) out of that...